### ISSUE TITLE
when "automatically hide scroll bars" is off in Windows, the content isn't pushed

### ISSUE THREAD

**Issue by Poopooracoocoo** [microsoft/Xaml-Controls-Gallery#439]
**Describe the bug**

when "automatically hide scroll bars" is off in Windows, the content isn't pushed and can make things tight

**Steps to reproduce the bug**

1. Enable "Automatically hide scroll bars in Windows"

**Expected behavior**

**Actual behavior**

**Screenshots**

![image](https://user-images.githubusercontent.com/51363864/82359087-c87b4280-9a4a-11ea-8bb7-881b6eb6fe68.png)
![image](https://user-images.githubusercontent.com/51363864/82359193-efd20f80-9a4a-11ea-8961-f68d2ada0904.png)

**Version Info**

Gallery version (found on Settings page): 

1.2.15.0

Windows 10 version:
- [ ] Insider Build (xxxxx)
- [x] May 2020 Update
- [x] November 2019 Update (18363)
- [ ] May 2019 Update (18362)
- [ ] October 2018 Update (17763)
- [ ] April 2018 Update (17134)
- [ ] Fall Creators Update (16299)
- [ ] Creators Update (15063)
- [ ] Anniversary Update (14393)

Device form factor:
- [x] Desktop
- [ ] Mobile
- [ ] Xbox
- [ ] Surface Hub
- [ ] IoT

**Additional context**

**Comment by chingucoding** [comment]
That is an issue with hierarchical NavigationView which is tracked here: https://github.com/microsoft/microsoft-ui-xaml/issues/2287

**Comment by Poopooracoocoo** [comment]
@chingucoding thanks. I think that it effects more than just the hierarchical NavigationView. I've noticed it in the print dialog and in the Microsoft Store.

**Comment by Poopooracoocoo** [comment]
and here's Inkodo's what's new dialog: 
![image](https://user-images.githubusercontent.com/51363864/82449112-0d0bea00-9aee-11ea-82b5-9815e2cf4f08.png)

**Comment by chingucoding** [comment]
I think this is something that should be discussed in the WinUI repo, since that's where any changes in that regard would be made.

**Comment by stmoy** [comment]
Agreed with @chingucoding. This is the default ScrollBar behavior that exists in the platform. If we are going to change this behavior, we should change it holistically. CC: @chigy
